src/compile/scale/range.ts

```diff
@@ -210,7 +225,16 @@ export function defaultRange(
       // TODO: support custom rangeMin, rangeMax
       const rangeMin = sizeRangeMin(mark, zero, config);
       const rangeMax = sizeRangeMax(mark, xyRangeSteps, config);
-      return [rangeMin, rangeMax];
+      if (isContinuousToDiscrete(scaleType)) {
+        // for now 4 is the default value for range cardinality. we might change it later
```

Comment:
Will "we really change it later"?  (Why we should we?)

In any case, the comment shouldn't be here?

Reply:
Done